modules/nw-sctp-verifying.adoc

```diff
@@ -0,0 +1,135 @@
+// Module included in the following assemblies:
+//
+// * networking/using-sctp.adoc
+
+:image: registry.access.redhat.com/ubi8/ubi
+
+ifdef::openshift-origin[]
+:image: fedora:31
+endif::[]
+
+[id="nw-sctp-verifying_{context}"]
+= Verifying Stream Control Transmission Protocol (SCTP) is enabled
+
+You can verify that SCTP is working on a cluster by creating a Pod with an application that listens for SCTP traffic, associating it with a Service, and then connecting to the exposed service.
+
+.Prerequisites
+
+* Access to the Internet from the cluster to install the `lksctp-tools` package.
+* Install the OpenShift Command-line Interface (CLI), commonly known as `oc`.
+* Access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Create a Pod that runs an SCTP listener:
+
+.. Create a file named `sctp-server.yaml` that defines a Pod using the following YAML:
++
+[source,yaml,subs="attributes+"]
+----
+apiVersion: v1
+kind: Pod
+metadata:
+  name: sctpserver
+  labels:
+    app: sctpserver
+spec:
+  containers:
+    - name: sctpserver
+      image: {image}
+      command: ["/bin/sh", "-c"]
+      args:
+       ["dnf install -y lksctp-tools && sctp_test -H localhost -P 30100 -l"]
+      ports:
+        - containerPort: 30100
+          name: sctpserver
+          protocol: SCTP
+----
+
+.. Create the Pod by entering the following command:
++
+----
+$ oc create -f sctp-server.yaml
```

Comment:
Pod not run when using above sctp-server.yaml due to socket: failed to create socket:  Socket type not supported
[weliang@weliang FILE]$ oc logs sctpserver
Updating Subscription Management repositories.
Unable to read consumer identity
Subscription Manager is operating in container mode.
This system is not registered to Red Hat Subscription Management. You can use subscription-manager to register.
Red Hat Universal Base Image 8 (RPMs) - BaseOS  3.5 MB/s | 761 kB     00:00    
Red Hat Universal Base Image 8 (RPMs) - AppStre  12 MB/s | 3.4 MB     00:00    
Red Hat Universal Base Image 8 (RPMs) - CodeRea  38 kB/s | 9.1 kB     00:00    
Dependencies resolved.
================================================================================
 Package            Architecture Version               Repository          Size
================================================================================
Installing:
 lksctp-tools       x86_64       1.0.18-3.el8          ubi-8-baseos       100 k

Transaction Summary
================================================================================
Install  1 Package

Total download size: 100 k
Installed size: 251 k
Downloading Packages:
lksctp-tools-1.0.18-3.el8.x86_64.rpm            1.1 MB/s | 100 kB     00:00    
--------------------------------------------------------------------------------
Total                                           1.0 MB/s | 100 kB     00:00     
Running transaction check
Transaction check succeeded.
Running transaction test
Transaction test succeeded.
Running transaction
  Preparing        :                                                        1/1 
  Installing       : lksctp-tools-1.0.18-3.el8.x86_64                       1/1 
  Running scriptlet: lksctp-tools-1.0.18-3.el8.x86_64                       1/1 
  Verifying        : lksctp-tools-1.0.18-3.el8.x86_64                       1/1 
Installed products updated.

Installed:
  lksctp-tools-1.0.18-3.el8.x86_64                                              

Complete!
local:addr=::1, port=rwp, family=10
seed = 1585331334

Starting tests...
	socket(SOCK_SEQPACKET, IPPROTO_SCTP)

		*** socket: failed to create socket:  Socket type not supported ***
[weliang@weliang FILE]$